cylc/flow/task_job_mgr.py

```diff
@@ -969,7 +969,7 @@ def _set_retry_timers(
 
         submit_delays = (
             rtconfig['submission retry delays']
-            or itask.platform['submission retry delays']
+            or itask.platform and itask.platform['submission retry delays']
```

Comment:
```suggestion
            or itask.platform.get('submission retry delays')
```

Reply:
What does `None.get` do? If the answer isn't safe then this is not right.